Makefile

```diff
@@ -302,7 +302,7 @@ appimageupdate: all
 	ln -sf ../../$(APPIMAGE_DIR)/AppRun $(INSTALL_DIR)/koreader
 	ln -sf ../../$(APPIMAGE_DIR)/koreader.appdata.xml $(INSTALL_DIR)/koreader
 	ln -sf ../../$(APPIMAGE_DIR)/koreader.desktop $(INSTALL_DIR)/koreader
-	ln -sf ../../resources/koreader.png $(INSTALL_DIR)/koreader
+	ln -sf ../../resources/koreader-square.png $(INSTALL_DIR)/koreader
```

Comment:
Hold on, that'll break things and it would necessitate further changes. I think it might be better to move it into platform/appimage/koreader.png